core/domain/stats_services.py

```diff
@@ -772,27 +921,38 @@ def get_exploration_stats_multi(exp_version_references):
     return exploration_stats_list
 
 
-def delete_playthroughs_multi(playthrough_ids):
+def delete_playthroughs_multi(playthrough_ids: List[str]) -> None:
     """Deletes multiple playthrough instances.
 
     Args:
         playthrough_ids: list(str). List of playthrough IDs to be deleted.
     """
 
     @transaction_services.run_in_transaction_wrapper
-    def _delete_playthroughs_multi_transactional():
+    def _delete_playthroughs_multi_transactional() -> None:
         """Implementation to be run in a transaction."""
-        stats_models.PlaythroughModel.delete_multi(
-            stats_models.PlaythroughModel.get_multi(playthrough_ids))
+        playthrough_models_with_none = stats_models.PlaythroughModel.get_multi(
```

Comment:
Same as above.

Should we change the variable name since `playthrough_model` is not `None`?

Reply:
If we keep the original name then at line 942 it will contradict the argument type because at line 942 `delete_multi` can only accept `non-none lists`.

Reply:
ditto as above

Reply:
I think we have already discussed this before (https://github.com/oppia/oppia/pull/15640#discussion_r913354208).

Reply:
We cannot change the name here, because we are not updating the model in any way instead we are just fetching the models and passing it to the methods.